src/config.js

```diff
@@ -1,6 +1,21 @@
 /* eslint-disable import/no-dynamic-require */
 /* eslint-disable global-require */
 const path = require('path')
+const joi = require('joi')
+
+
+const environmentSchema = joi.object().keys({
+	DATABASE_URL: joi.string().required(),
+	PORT: joi.number().required(),
+	TOKEN_SECRET: joi.string().required(),
+	EMAIL_SERVICE_HOST: joi.string().required(),
+	EMAIL_SERVICE_PORT: joi.number().required(),
+	EMAIL_SERVICE_PASSWORD: joi.string().required(),
+	EMAIL_SERVICE_USERNAME: joi.string().required(),
+	EMAIL_FROM: joi.string().required(),
```

Comment:
now seeing these, don't mind me. i think there should be a way to check for an email as well

Reply:
yes, currently working on it. @jocrah